javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,47 @@
+//GIPHY
+document.addEventListener("DOMContentLoaded", giphyInit);
+
+function giphyInit() {
+  const btnSearch = document.querySelector(".btnSearch");
+  btnSearch.addEventListener("click", (ev) => {
+    ev.preventDefault(); // to stop the page reload
+    let numGif = document.querySelector(".num").value;
+    console.log("num:" + numGif);
+
+    let url = `http://api.giphy.com/v1/gifs/search?q=ryan+gosling&api_key=FtR8NF0BZTC8WREmYfUfvQYzxO8uGHBA&limit=${numGif}&q=`;
```

Comment:
Here I can't see input search values, I only see input values to limit the number of Gif displays. Do you work only on "ryan+gosling" gifs?

Reply:
As Hailegebriel22 commented, you need to add the apiKey and searchedWord and a limitvalue to url when you do fetch url.

like this:

fetch(`https://api.giphy.com/v1/gifs/search?api_key=${apiKey}&q=${searchedWord}&limit=${gifLimit}&offset=0&rating=g&lang=en`)